driver-core/src/test/resources/unified-test-format/command-monitoring/pre-42-server-connection-id.json

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
This test is skipped entirely in code, but per previous conversation better to still include it in the repository